src/sympc/protocol/fss/fss.py

```diff
@@ -124,7 +126,19 @@ def fss_op(x1: MPCTensor, x2: MPCTensor, op="eq") -> MPCTensor:
     Returns:
         MPCTensor: Shares of the comparison.
     """
-    assert not th.cuda.is_available()  # nosec
+    if th.cuda.is_available():
+        # FSS is currently not supported on GPU.
+        # https://stackoverflow.com/a/62145307/8878627
+        try:
+            cuda_visible_devices = os.environ["CUDA_VISIBLE_DEVICES"]
```

Comment:
MAYBE: Use `os.environ.get("CUDA_VISIBLE_DEVICES", None)``` and remove the except block

Reply:
Great! Never knew of it's existence.

Reply:
Done! :partying_face: 
Nice tip of the `.get()`. I'll remember it :smile: :hugs: